### DISCUSSION TITLE
Relative URLs in the browseable view are not clickable

### DISCUSSION THREAD

**Discussion by alessandrosp** [encode/django-rest-framework#8460] [Potential Issue] [answered]
As per the title. URLs that are relative are not clickable in the browseable interface.

**Comment by alessandrosp** [comment, marked as answer]
There's an easy fix. Just add a `or value.startswith('/')` here: https://github.com/encode/django-rest-framework/blob/71e6c30034a1dd35a39ca74f86c371713e762c79/rest_framework/templatetags/rest_framework.py#L221

**Reply by lovelydinosaur** [reply]
Now resolved via your pull request #8464 - thanks.
